paper/paper.md

```diff
@@ -31,122 +31,109 @@ affiliations:
+The object `Semidiscretization` couples the systems of a simulation and also creates the corresponding neighborhood searches for each system. The resulting ODE problem is then fed into the time integrator and is solved by an appropriate numerical time integration scheme.
 
+![Main building blocks of TrixiParticles.jl \label{fig:structure}](structure.png){width=100%}
 
-![TODO: caption *Inspired by [docs Trixi](https://trixi-framework.github.io/Trixi.jl/stable/overview/#overview-semidiscretizations)* \label{fig:structure}](structure.png){width=100%}
+The code presented here can be found on [GitHub](https://github.com/trixi-framework/TrixiParticles.jl),  along with a detailed [manual](https://trixi-framework.github.io/TrixiParticles.jl/stable/) explaining how to use the package. Additionally, we provide tutorials explaining how to set up a simulation for fluids, structure mechanics or a fluid-structure interaction.
+A collection of predefined simulation files can be found in the `examples` directory.
 
+### Feature highlights
 
-At present, TrixiParticles.jl includes the implementation of the following systems
+So far, the following feature highlights have been implemented:
 
 * *Fluid Systems*
-    + [`WeaklyCmpressibleSPHSystem` (WCSPH)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/weakly_compressible_sph/): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
-    + [`EntropicallyDampedSPHSystem` (EDAC)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/entropically_damped_sph/): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
+    + `WeaklyCmpressibleSPHSystem` (WCSPH): Standard SPH method originally developed by [@Monaghan:1977] to simulate astrophysics applications.
+    + `EntropicallyDampedSPHSystem` (EDAC): As opposed to the WCSPH scheme, which uses an equation of state, this scheme uses a pressure evolution equation to calculate the pressure which is derived by [@Clausen:2013] and adapted to SPH by [@Ramachandran:2019].
 
 * *Structure Systems*
-    + [`TotalLagrangianSPHSystem` (TLSPH)](https://trixi-framework.github.io/TrixiParticles.jl/stable/systems/total_lagrangian_sph/): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
+    + `TotalLagrangianSPHSystem` (TLSPH): System to simulate elastic structure where the interaction is defined with stress tensors [@O_Connor:2021].
     + `DEMSystem`: Discrete element system [@Bicanic:2004], [@Cundall:1979].
```

Comment:
Should we elaborate a little more here?

Reply:
word count (~1100) says no :D

Reply:
Maybe just "Discrete element system for granular flows" or so?